packages/fresh_dio/test/fresh_test.dart

```diff
@@ -130,7 +130,7 @@ void main() {
         expect(
           (result.captured.first as RequestOptions).headers,
           {
-            'content-type': 'application/json; charset=utf-8',
+            // 'content-type': 'application/json; charset=utf-8',
```

Comment:
why are these lines commented out? If there are no longer needed can we remove them instead?

Reply:
Content-type is implied as show in [Dio README.md](https://github.com/cfug/dio/blame/main/dio/README.md#L317).
The pretty_dio_logger showed me this:
`
flutter: ╔ Headers
flutter: ╟ contentType: null
flutter: ╟ responseType: ResponseType.json
`

More info can be seen in this [commit](https://github.com/cfug/dio/commit/cc54ef4c6ecf113b574e321034b595b6f50c4976).